rclcpp/include/rclcpp/parameter.hpp

```diff
@@ -93,6 +95,30 @@ class ParameterVariant
 
   inline ParameterType get_type() const {return static_cast<ParameterType>(value_.type); }
 
+  inline std::string get_type_name() const
+  {
+    switch (get_type()) {
+      case rclcpp::parameter::ParameterType::PARAMETER_BOOL:
+        return "bool";
+        break;
+      case rclcpp::parameter::ParameterType::PARAMETER_INTEGER:
+        return "integer";
+        break;
+      case rclcpp::parameter::ParameterType::PARAMETER_DOUBLE:
+        return "double";
+        break;
+      case rclcpp::parameter::ParameterType::PARAMETER_STRING:
+        return "string";
+        break;
+      case rclcpp::parameter::ParameterType::PARAMETER_BYTES:
+        return "bytes";
+        break;
+      case rclcpp::parameter::ParameterType::PARAMETER_NOT_SET:
+      default:
```

Comment:
Separate another case where the same exception was used: 3b337fd98091bfb4853f81979c178db3b0b4f59a